numbat/modules/core/datetime.nbt

```diff
@@ -0,0 +1 @@
+fn now() -> DateTime
```

Comment:
If we do `numbat --no-prelude`, and then only import `use core::datetime`, we don't have a `Time` dimension. So doing something like
```
use core::datetime
now() - now()
```
will crash.

Could be resolved for now by importing `core::dimensions`?


Minor nitpick: add a trailing newline?

Reply:
Sure, I'll try that.  Also, I intend to have some type of nice error message like "a Time dimension was not found" as a hint for people who are building their own custom preludes.

Reply:
Unfortunately importing `core::dimensions` is not sufficient, because `Seconds` are needed, so I'm importing `units::si`.  It feels wrong for something in `core` to import something in `units`, though...